sound/soc/sof/intel/tgl.c

```diff
@@ -50,6 +50,24 @@ static int tgl_dsp_core_put(struct snd_sof_dev *sdev, int core)
 	return 0;
 }
 
+static int tgl_ipc4_dsp_core_put(struct snd_sof_dev *sdev, int core)
```

Comment:
commit message: all acronyms need to be in capital letters, that's IPC4, IMR, IPC3.

Do no abbreviate fw, use firmware. Also use shorter sentences without combinations of but/and.

Reply:
sure, thanks